tensorflow_blade/docs/tutorials/tensorflow_blade_bert_inference.md

```diff
@@ -0,0 +1,110 @@
+# Tutorial: Optimize BERT Inference with Tensorflow-Blade
+
+In this tutorial, we show how to optimize BERT model for inference with a few lines code to call tensorrt optimization pass from tensorflow-blade.
+
+**BERT**, short for Bidirectional Encoder Representations from Transformers, is one of the most popular natural language processing (NLP) model in the world. BERT models and its many varieties have been widely used for language modelling tasks. **However, with great model accuracy comes with a great amount of computations.** The latency of BERT model is quite high, since it has both massive GEMM operations and element-wise operations with lots of redundancy memory accesses, making BERT model difficult to deploy for real-time applications.
+In this tutotial, we use a pre-trained "**bert-base-cased**" BERT model from **Huggingface Transformers**.
+
+**TensorRT** is an SDK for deep learning inference powered by Nvidia, which includes a deep learning inference optimizer and runtime that delivers low latency and high throughput for inference applications.
```

Comment:
Key logic missing: why not use TensorRT directly?

Reply:
I think maybe we should explains this in the design of tf2trt part, and reference here will be ok.

Reply:
At least it should be explained here why `trtexec` is unable to optimize this model? Or `trtexec` is ok ? -_-!